_sass/layout/_navigation.scss

```diff
@@ -110,6 +104,7 @@
 
       .nav-list-item {
         position: relative;
+        min-height: 48px;
```

Comment:
I think this gives too much height on secondary nav items. I'd remove it and use something like `padding: 4px 0;`